### ISSUE TITLE
Support for ObjectMapper v0.18

### ISSUE THREAD

**Issue by PranKe01** [tristanhimmelman/AlamofireObjectMapper#37]
AlamofireObjectMapper v1.0 does not support the current version of ObjectMapper v0.18.

**Comment by patricks** [comment]
+1

**Comment by RodolfoGS** [comment]
I have this error when run `carthage update`

```
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
CodeSign error: code signing is required for product type 'Framework' in SDK 'iOS 9.0'
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by tristanhimmelman** [comment]
I have released a new version (1.1.0) with support for the latest version of ObjectMapper
